package.json

```diff
@@ -2,9 +2,10 @@
   "name": "electron-pdf",
   "version": "0.12.0",
   "description": "A command line tool to generate PDF from URL, HTML or Markdown files",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "standard"
+    "lint": "standard",
+    "test": "standard && node_modules/.bin/electron node_modules/.bin/jasmine"
```

Comment:
If the `index.js` is expected to be run inside of an electron process, I have this covered here http://webuild.envato.com/blog/running-headless-javascript-testing-with-electron-on-any-ci-server/